src/utils/extractFrameMetaTags.ts

```diff
@@ -0,0 +1,22 @@
+export async function extractFrameMetaTags(
+  url: string,
+): Promise<Record<string, string>> {
+  try {
+    const res = await fetch(url)
+    const text = await res.text()
+
+    const match = text.matchAll(/<meta property="fc:(\S+)" content="(\S+)"/gm)
```

Comment:
let's also match on [`frog:version`](https://github.com/wevm/frog/blob/ace53468e9ea370a6d0225fc59f510692ba8bdc3/src/frog-base.tsx#L390) too if we can. it's basic meta tag to help with debugging frames in the wild.

Reply:
done